src/bundle.ts

```diff
@@ -0,0 +1,109 @@
+import * as path from "path";
+import { SourceNode } from "source-map";
+import * as ts from "typescript";
+import { couldNotFindBundleEntryPoint } from "./diagnostics";
+import { EmitHost, TranspiledFile } from "./Transpile";
+import { formatPathToLuaPath, trimExtension } from "./utils";
+import { escapeString } from "./TSHelper";
+import { CompilerOptions } from "./CompilerOptions";
+
+const formatPath = (path: string) => escapeString(formatPathToLuaPath(trimExtension(path)));
+const modulePath = (baseDir: string, pathToResolve: string) => formatPath(path.relative(baseDir, pathToResolve));
```

Comment:
I think that name sounds too variabl-y (`moduleSourceNode` already has a parameter with the same name). Also, conceptually it's close to `getExportPath` from tshelper, which is used only in tests now. Maybe we can just update it's implementation and use it instead?